app/components/geoblacklight/header_component.html.erb

```diff
@@ -0,0 +1,18 @@
+<%= top_bar %>
+<% if controller_name == 'catalog' && (helpers.has_search_parameters? || params[:action] == 'show') %>
```

Comment:
It would be nice to put this into a method like `show_search_bar?`

Reply:
agreed.